Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson4
+    {
+        public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
+                var name = GetString("What is your name: ");
+                var surname = GetString("What is your surname: ");
+                var age = GetInt("What is your age: ");
+
+
+                var weight = GetFloat("What is your weight in kilogram: ");
+                var height = GetFloat("What is your height in metres: ");
+                var bmi = GetBmi(weight, height);
+
+                Console.Write($"{name}  {surname}  is {age} years old, his weight is {weight} kg and his height is {height:F2} cm.");
+                Console.WriteLine($"His BMI is:  {bmi:N2}");
+                Console.WriteLine("");
+                if (i == 0)
+                {
+                    Console.WriteLine($"Now for another person {Environment.NewLine}  ");
+                }
+            }
+        }
+
+        public static float GetBmi(float weight, float height)
+        {
+            if (height <= 0)
+            {
+                Console.WriteLine($"Height cannot be equal or less than zero, but was {height}."); 
+            }
+
+            if (weight <= 0)
+            {
+                Console.WriteLine($"Weight cannot be equal or less than zero, but was {weight}.");
+            }
+
+
+            return weight / (float)Math.Pow(height, 2);
+        }
+
+        public static string GetString(string message)
+        {
+            Console.WriteLine(message);
+            var input = Console.ReadLine();
+            const int ERRORCODE_INT = -1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'ERRORCODE_INT'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4877841)